kms/src/main/java/com/example/CreateKeyAsymmetricDecrypt.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
```

Comment:
We recently changed the rule around this to [encourage shorter package names](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#package-names):
```suggestion
package kms;
```

Please also update the `groupId` in the `pom.xml`.

Reply:
I don't understand this comment. Which pom.xml? I don't see com.example defined in the XML, and changing the package name makes everything fail to compile.

Reply:
Ah I have to move the files. Done.